tests/test_show.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+from pypst import Document, Heading, Show
+
+
+def test_show_rule_no_arg():
+    obj = Show(selector="heading", body="text.with(fill: red)")
+    assert obj.render() == "#show heading: text.with(fill: red)"
+
+
+def test_show_rule():
+    obj = Show(selector="heading", argument="it", body="text(fill: red, it)")
+
+    assert obj.render() == "#show heading: it => text(fill: red, it)"
```

Comment:
Test cases with `Content` and `Functional` objects as body are missing. They are explicitly checked against in the post-init function and should therefore have test cases.